src/main/scala/leon/datagen/VanuatooDataGen.scala

```diff
@@ -41,16 +41,23 @@ class VanuatooDataGen(ctx: LeonContext, p: Program) extends DataGenerator {
     (n, d) -> Constructor[Expr, TypeTree](List(), RealType, s => FractionalLiteral(n, d), "" + n + "/" + d)
   }).toMap
 
+  val strings = (for (b <- Set("", "a", "\"\t\n", "Abcd")) yield {
```

Comment:
This example containing "\t\n seems rather arbitrary and is a bit weird no? Is this specific to the kind of strings you want to generate? I think it makes more sense to have other more common string literal, like "", "a", "foo", "bar".

If you want it to generate special strings you can always overwrite this and generate weird strings